### DISCUSSION TITLE
Reset container size

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5943] [Q&A] [answered]
Image that I have two containers open in a workspace. I can resize these containers with $mod+r and arrow keys. However, after resizing a container in this way, to return to its original size I have to use $mod+r and arrow keys again.

If I resize one of these containers, how do I get it to return to its original size (both occupying 50% of the screen) with just a command/shortcut? Is this possible in i3wm? Searching the documentation I didn't find anything that indicated this possibility.

**Comment by okraits** [comment, marked as answer]
The following command works for me:

`resize set width 50 ppt height 50 ppt`

In the userguide: https://i3wm.org/docs/userguide.html#resizingconfig

**Reply by ghost** [reply]
Oh, of course! I was trying to use height and width separately. It would never work! That's why I thought there was no way. Thanks, my mistake.

Next time I will read the documentation more carefully! :)
